tests/System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -1224,7 +1228,7 @@ public void MockDirectory_GetDirectories_RelativeDirectory_WithChildren_ShouldRe
 
             // Assert
             CollectionAssert.AreEqual(
-                new[] { XFS.Path(currentDirectory + @"\" + relativeDirPath + @"\child") },
+                new[] { XFS.Path(relativeDirPath + @"\child") },
```

Comment:
I think the current implementation does not work for deeper directory structures, e.g. if you have a `child/grandchild` directory here. Could you please add some tests for such scenarios?